### DISCUSSION TITLE
Issue

### DISCUSSION THREAD

**Discussion by transgirllucy** [searx/searx#2818] [Installation issue: custom]
searx@vps2148105:~/searx-src$ python searx/webapp.py 
  File "searx/webapp.py", line 966
    nonlocal total_length
                        ^
SyntaxError: invalid syntax

**Comment by Alex-Programs** [comment]
I also have this issue. It seems to occur when you don't run as the searx user; I would run as the searx user, but PM2 doesn't support it.

**Comment by 0xhtml** [comment]
The error probably has to do with the version of python. Try running `python3 searx/webapp.py`. Maybe that fixes the error.
